README.md

```diff
@@ -6,12 +6,15 @@
 Unified Workflow tools for use with applications with UFS and beyond
 
 ## Installation
-Simple installation instructions
+Simple installation instructions with a conda environment.
 ```sh
 $> git clone https://github.com/ufs-community/workflow-tools
+$> conda env create -f environment.yaml
+$> conda activate workflow_tools
 $> cd workflow-tools
-$> pip install .
+$> export PYTHONPATH=<workflow-tools location>/src:$PYTHONPATH
```

Comment:
I think we could direct folks to use this syntax so that we can just have them copy/paste:
```suggestion
$> export PYTHONPATH=$(pwd)/src:$PYTHONPATH
```

Reply:
Ah, much cleaner - I thought about changing the read-the-docs, but thats better to leave with all the detail I think